pkgs/by-name/ss/sscep/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gitUpdater,
+  openssl,
+  autoconf,
+  automake,
+  libtool,
+  pkg-config,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sscep";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "certnanny";
+    repo = "sscep";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-wlxQONOCLPuNdI6AyMJoLP09cs+ak7Jv9idhXTT5RWA=";
+  };
+
+  preConfigure = ''
+    ./bootstrap.sh
+  '';
+
+  buildInputs = [ openssl ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
```

Comment:
`autoReconfHook` I guess? Also is `bootstrap.sh` necessary?

Reply:
Resolved with Resolved with 388e97119f9345968fd16f96ecd1dc3968136677